sdks/python/src/opik/decorator/base_track_decorator.py

```diff
@@ -418,6 +418,9 @@ def _before_call(
             context_storage.set_trace_data(created_trace_data)
             TRACES_CREATED_BY_DECORATOR.add(created_trace_data.id)
 
+            client = opik_client.get_client_cached()
+            client.trace(**created_trace_data.trace_start_parameters)
```

Comment:
As mentioned in another comment, this place should contain if-condition to decide - log or not.